cmd/nerdctl/network_rm_linux_test.go

```diff
@@ -49,3 +50,24 @@ func TestNetworkRemove(t *testing.T) {
 	_, err = netlink.LinkByName("br-" + networkID[:12])
 	assert.Error(t, err, "Link not found")
 }
+
+func TestNetworkRemoveWhenLinkWithContainer(t *testing.T) {
+	t.Parallel()
+	if rootlessutil.IsRootless() {
+		t.Skip("test skipped for remove rootless network")
```

Comment:
Why?

Reply:
Keep the same with previous network_rm test